### PR TITLE
Added Mk2 wrist folder

### PR DESCRIPTION
**Pull request by ale-git**
Contains:
- Simmechanics model
- Simulink control system configured for decoupler embedded code generation
- Mathematics of spherical wrist documentation

### CODE REVIEW

File: wrist_mk2/README.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+It seems impossible to find the **analytic solution** in closed form of the *direct geometric problem* (i.e. given the motor angles, find the orientation of the platform), but it is quite easy to calculate the analytic form of the inverse Jacobian if we express the problem in a smart way. The **numeric solution** of the *direct geometric problem* is quite easy as well.
+
+## Premise
+
+-   The platform rotates without translation around the origin $O$.
+-   The length of the arms, equal to the platform radius, is normalized as 1.
+-   Let's call $q_i$ the positions of the joints on the platform and $p_i$ the positions of the joints in the arms.
+-   The "L" shape of the connetcting rods forces each vector $q_i$ to be always orthogonal to its respective vector $p_i$ in all configurations.
+-   The euclidean distance $L$ between the $q_i$ and the respective $p_i$ is imposed to be constant by the connecting rod.
+-   In the case of symmetrical structure, the coordinates of the platform joints in the rest configuration are $q_1=(1,0,0)$, $q_2=(-\frac{1}{2},\frac{\sqrt{3}}{2},0)$ e $q_3=(-\frac{1}{2},-\frac{\sqrt{3}}{2},0)$
```
Comment (pattacini) [inline wrist_mk2/README.md:16]:
```suggestion
-   In the case of symmetrical structure, the coordinates of the platform joints in the rest configuration are $q_1=(1,0,0)$, $q_2=(-\frac{1}{2},\frac{\sqrt{3}}{2},0)$ and $q_3=(-\frac{1}{2},-\frac{\sqrt{3}}{2},0)$
```

File: wrist_mk2/platform_wris_-60_50_prt.step
Comment (pattacini) [inline wrist_mk2/platform_wris_-60_50_prt.step:1]: the name seems missing the ending `t`.

**Review by pattacini — CHANGES_REQUESTED** [review]
Nice work @ale-git, especially the description of the underlying maths.

I've got a couple of suggestions:
- It'd be nice to segregate all the step files into a dedicated subfolder.
- Having a MATLAB project to click on to get everything started is a very convenient practice. This way, one already knows what to do when landing here.
- The README should also contain a brief section on code generation, especially for what regards the difference between the left and the right wrist.

**Comment by pattacini** [comment]
This is a PR relevant to @marcoaccame @sgiraz @mfussi66.

/remind in 7 days to clean up our internal repo of physical models

**Comment by ale-git** [comment]
I'm closing and reastarting since the workflow is messed up because of missing lfs management.

**Comment by pattacini** [comment]
Follow-up in #26

**Pull request closed without merging**